base/server/upgrade/11.1.0/03-RemoveStaticInfoInServerXML.py

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright Red Hat, Inc.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+from __future__ import absolute_import
+import logging
+
+import pki.server.upgrade
+
+logger = logging.getLogger(__name__)
+
+
+class RemoveStaticInfoInServerXML(
+        pki.server.upgrade.PKIServerUpgradeScriptlet):
+
+    def __init__(self):
+        super(RemoveStaticInfoInServerXML, self).__init__()
```

Comment:
In `Python 3` it is no longer necessary to pass the current class and instance in `super()`; the new implementation of `super()` assumes the current class and instance as the default so this could be `super().__init__()`

Reply:
Thanks! It's a lot cleaner.